sdks/python/apache_beam/yaml/tests/create.yaml

```diff
@@ -81,3 +81,37 @@ pipelines:
               - {sdk: MapReduce, year: 2004}
               - {sdk: Flume}
               - {sdk: MillWheel, year: 2008}
+
+  # Simple Create with explicit null value
+  - pipeline:
+      type: chain
+      transforms:
+        - type: Create
+          config:
+            elements:
+              - {sdk: MapReduce, year: 2004}
```

Comment:
@damccorm this is related to what we discussed. We already have `ValidateWithSchema` for YAML, which can help validate the schema. But we do not have a way to define the schema. For example, here, year could be NULL but the inferred schema from Beam is Optional[Any], which cannot be accepted by WriteToBigQuery since WriteToBigQuery does not allow any type. We should do something like `ValidateWithSchema` that allows users to define the output schema.

Reply:
Discussed with @damccorm offline and he thought it was a good idea with adding an output schema to create and most other transforms with bad records going to error handling.  I will open an issue for this.  Thanks.